docs/source/reference/indexing.rst

```diff
@@ -96,6 +96,7 @@ MultiIndex Properties
    MultiIndex.names
    MultiIndex.ndim
    MultiIndex.T
+   MultiIndex.nlevels
```

Comment:
But this addition seems good to me.

i think we can merge this PR with this only, except in `indexes.py`